patch.diff

```diff
@@ -0,0 +1,15302 @@
+From 4c71e13695a1b18cd76de6f340ca023ea5ca4399 Mon Sep 17 00:00:00 2001
```

Comment:
@srottem is this file redundant?
If yes - please remove it from history (rm, amend commit + force push)

Reply:
Removed - apologies.

Reply:
thanks